plugins/network-policy-audit/tests/integration/12_run_tc_inv_007.sh

```
@@ -0,0 +1,54 @@
+    matchLabels:
+      app: postgres
+  policyTypes:
+  - Ingress
+  ingress:
+  - from:
+    - podSelector:
+        matchLabels:
+          app: backend
+EOF
+
+cd ${PLUGIN_DIR}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)